{{cookiecutter.project_name}}/setup.cfg

```diff
@@ -29,7 +29,7 @@ collect_ignore = ['setup.py']
 
 [coverage:report]
 show_missing = true
-fail_under = 65
+fail_under = 70
```

Comment:
Why does this increase in this PR?

Reply:
Yeah, not needed necessarily here. Same goes for the spellcheck errors

Reply:
Removed it